sympy/core/relational.py

```diff
@@ -443,17 +443,19 @@ def _eval_simplify(self, **kwargs):
             if len(free) == 1:
                 try:
                     from sympy.solvers.solveset import linear_coeffs
+                    from typing import Optional
```

Comment:
I don't think this is necessary (at least here). What is the purpose?

Reply:
Are you asking what is the purpose of  static types in programming languages?  Here it's useful to indicate that the bool is *not* two-valued.

Reply:
Although the topic about necessity static typing is controversial, and I am one of a few people who is actually pro for static typing, 

however, importing inside function is also another question.
I think that you should move the import to the top. Importing inside function is only necessary to avoid cyclic dependency.

Reply:
And also, I think that the effort like `mz: Optional[bool] = m.is_zero` may not be useful, than annotating `m.is_zero`.
There is already an ongoing effort for this https://github.com/sympy/sympy/pull/25103

Also, to add some information, you can see some influential Python developers suggesting to use `| None` than `Optional`
https://discuss.python.org/t/clarification-for-pep-604-is-foo-int-none-to-replace-all-use-of-foo-optional-int/26945/9